src/test/java/org/openstreetmap/atlas/utilities/command/subcommands/AtlasShellToolsDemoCommandTest.java

```diff
@@ -0,0 +1,37 @@
+package org.openstreetmap.atlas.utilities.command.subcommands;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.utilities.collections.Maps;
+
+/**
+ * @author lcram
+ */
+public class AtlasShellToolsDemoCommandTest
+{
+    @Test
+    public void testCommand()
+    {
+        final ByteArrayOutputStream outContent1 = new ByteArrayOutputStream();
+        final ByteArrayOutputStream errContent1 = new ByteArrayOutputStream();
+        final InputStream inStream = new ByteArrayInputStream("hello!".getBytes());
```

Comment:
Here you could use `new StringInputStream("hello!")` but that does not matter really. Just FYI